common/test/acceptance/tests/helpers.py

```diff
@@ -215,6 +216,22 @@ def select_option_by_value(browser_query, value):
     select = Select(browser_query.first.results[0])
     select.select_by_value(value)
 
+    def options_selected():
+        """
+        Returns True if all options in select element where value attribute
+        matches `value`. if any option is not selected then returns False
+        and select it.
+        """
+        all_options_selected = True
```

Comment:
As written, if the thing you wanted to select is not an option choice, then this method will return True. I think you need to account for that case also (e.g. the option list could be populated/updated by an ajax call).